tests/workers/test_geventlet.py

```diff
@@ -2,6 +2,11 @@
 #
 # This file is part of gunicorn released under the MIT license.
 # See the NOTICE for more information.
+import pytest
 
+
+@pytest.mark.xfail(
+    reason="TypeError: cannot set 'is_timeout' of immutable type 'TimeoutError'"
+)
```

Comment:
`eventlet==0.33.0` released py310 specific workarounds for this in https://github.com/eventlet/eventlet/commit/15d3c2477510229e1b49c60fccfde5ffe94b2a76, so it's no-longer required.  Test passes for me now on `py310`, so let's drop this change.